app/models/manageiq/providers/vmware/infra_manager/refresh_worker/runner.rb

```diff
@@ -12,9 +12,8 @@ def before_exit(_message, _exit_code)
     stop_inventory_collector if ems.supports_streaming_refresh?
   end
 
-  def message_delivery_suspended?
-    # If we are using streaming refresh don't dequeue EmsRefresh queue items
-    ems.supports_streaming_refresh? || super
+  def do_before_work_loop
+    super unless ems.supports_streaming_refresh?
```

Comment:
If we queue an initial full refresh then we end up doing two fulls, one on the initial startup and one when the queue msg gets processed.